frontend/src/Utils/convertToNMP.ts

```diff
@@ -35,7 +34,7 @@ const convertToNMP = (
   const newFields: NmpFieldInterface[] = newDetails.Fields.map((field: FieldDetailInterface) => {
     const newCrops: NmpCropInterface[] = field.Crops.map((crop: CropsDetailsInterface) => ({
       ...templateCropNMP,
-      id: crop.id + 1,
+      id: crop.id,
```

Comment:
Tried matching our index to nmp... bad idea and it still loads fine, so better leave it as was before this addition, indexed at 0.
This conversion happens multiple times through the calculation, which compounded this addition.